quarkus-build-caching-extension/src/main/java/com/gradle/QuarkusExtensionConfiguration.java

```diff
@@ -0,0 +1,106 @@
+package com.gradle;
+
+import org.apache.maven.project.MavenProject;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.util.Properties;
+
+final class QuarkusExtensionConfiguration {
+
+    // Environment variable key to disable caching
+    private static final String GRADLE_QUARKUS_KEY_CACHE_ENABLED = "GRADLE_QUARKUS_CACHE_ENABLED";
```

Comment:
Can we prefix these all with `DEVELOCITY` instead of `GRADLE`?

Reply:
Sure